.ci/azure/pipelines/obs-packaging-ci.yml

```diff
@@ -11,9 +11,37 @@ pool:
   vmImage: 'ubuntu-latest'
 
 steps:
-- script: echo setup
-  displayName: 'setup CI'
+# azure docker installation task
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/docker-installer?view=azure-devops 
+- task: DockerInstaller@0
+  displayName: 'Install Docker'
 
-- script: |
-    echo This is an initial CI job
-  displayName: 'Run CI'
+- bash: |
+   sudo apt-get update
+   sudo apt-get install -y git
+   git config --global user.email "azure-pipeline@kata.io"
+   git config --global user.name "azure-pipeline"
+  displayName: 'Setup'
```

Comment:
the `displayName` is going with the code above it is it? I wonder if you can move them to the front of the sections? I think that would read more clearly?

Reply:
It has to be indented to the `bash` ,  I think I can not move it.